src/uct/ib/mlx5/rc/rc_mlx5_iface.c

```diff
@@ -31,6 +31,7 @@ enum {
      * compatibility */
 };
 
+static const char *tm_type_to_str[2] = {"basic", "tag matching"};
```

Comment:
pls move it inside uct_rc_mlx5_iface_is_reachable_v2 function
and can remove "2" and just use "[]"